tools/kubectl-karpenter/pkg/kubectl-karpenter/kubectl-karpenter.go

```diff
@@ -0,0 +1,161 @@
+package kubectlkarpenter
+
+import (
+	"fmt"
+
+	"github.com/aws/karpenter/pkg/apis"
+	"github.com/aws/karpenter/pkg/apis/v1alpha1"
+	"github.com/aws/karpenter/pkg/apis/v1beta1"
+	nodeclassutil "github.com/aws/karpenter/pkg/utils/nodeclass"
+	"github.com/spf13/cobra"
+
+	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
+	corev1beta1 "github.com/aws/karpenter-core/pkg/apis/v1beta1"
+	nodepoolutil "github.com/aws/karpenter-core/pkg/utils/nodepool"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"k8s.io/cli-runtime/pkg/resource"
+	"k8s.io/kubernetes/pkg/printers"
+
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+type Options struct {
```

Comment:
Same question here about the options struct. High-level, what's the value in all of these members and do we need all of these things right now?

Reply:
This is more a context rather than options. I've renamed it to make it clear.